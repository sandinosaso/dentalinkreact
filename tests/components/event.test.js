import React from 'react';
import { shallow } from 'enzyme';

import Event from '../../js/components/event';

describe('<Event />', () => {
  it('renders an Event component', () => {
    const event = { start_at: '2016-05-16 08:00:00', due_at: '2016-05-16 10:00:00', name: 'Event 1' };
    const wrapper = shallow(<Event data={event} />);
    expect(wrapper).toMatchSnapshot();
  });
});
