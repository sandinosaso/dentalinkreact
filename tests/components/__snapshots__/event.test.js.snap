// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Event /> renders an Event component 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Event
    data={
      Object {
        "due_at": "2016-05-16 10:00:00",
        "name": "Event 1",
        "start_at": "2016-05-16 08:00:00",
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        "08:00",
        "-",
        "10:00",
      ],
      "className": "event",
      "style": Object {
        "height": "undefinedpx",
        "left": "undefined%",
        "top": undefined,
        "width": "undefined%",
      },
    },
    "ref": null,
    "rendered": Array [
      "08:00",
      "-",
      "10:00",
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          "08:00",
          "-",
          "10:00",
        ],
        "className": "event",
        "style": Object {
          "height": "undefinedpx",
          "left": "undefined%",
          "top": undefined,
          "width": "undefined%",
        },
      },
      "ref": null,
      "rendered": Array [
        "08:00",
        "-",
        "10:00",
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFifteenAdapter {
      "options": Object {
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
